.github/workflows/pr-release.yaml

```
@@ -8,7 +8,7 @@ jobs:
     name: "Release PR to npm"
     runs-on: ubuntu-latest
     # comment out if:false to enable release PR to npm
-    if: false
+    # if: false
```

```suggestion
    if: false
```

Confirmed working in browser